fendermint/vm/interpreter/src/fvm/gas/mod.rs

```diff
@@ -0,0 +1,19 @@
+// Copyright 2022-2024 Protocol Labs
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use fendermint_actor_gas::Gas;
+
+pub(crate) mod default;
+
+/// Handles the gas modeling in the current blockchain
+pub trait GasLayer {
+    /// The state of the blockchain
+    type State;
+
+    /// Update the block gas limit. This will only take effect in the next block.
+    #[allow(dead_code)]
+    fn set_block_gas_limit(&self, state: &mut Self::State, limit: Gas) -> anyhow::Result<()>;
```

Comment:
I was thinking of a way to let the admin call this function to update block gas limit. But I think it works without this method as well, since one can just call the actor